roofit/hs3/CMakeLists.txt

```
@@ -0,0 +1,49 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT rootfit/histfactory package
+# @author Carsten Burgard, CERN
+############################################################################
+
+ROOT_STANDARD_LIBRARY_PACKAGE(RooFitHS3
+  HEADERS
+  JSONInterface.h
+  RooJSONFactoryWSTool.h
+  HistFactoryJSONTool.h
+  JSONParser.h
+  RYMLParser.h
```

```suggestion
    RooFitHS3/JSONInterface.h
    RooFitHS3/RooJSONFactoryWSTool.h
    RooFitHS3/HistFactoryJSONTool.h
    RooFitHS3/JSONParser.h
    RooFitHS3/RYMLParser.h
```

In newer RooFit code (e.g. RooStats, HistFactory, and new multiprocessing libraries), we put the readers in a subdirectory of `inc`. Please move the header files and adapt the CMakeLists.txt (also fix the indentation please).